### PR TITLE
VORTEX 5676 : Center on any item does not center on it.

### PR DESCRIPTION
**Pull request by brelsftm**
Fixes #

Centering on a feature(s) works correctly in 2d mode ( Equirectangular ) .

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: open-sphere-base/core/src/main/java/io/opensphere/core/viewer/impl/DynamicViewer.java [hunk 1]
```diff
@@ -60,6 +60,20 @@
      */
     double getHeading();
 
+    /**
```
Comment (roddyaj) [inline open-sphere-base/core/src/main/java/io/opensphere/core/viewer/impl/DynamicViewer.java]: I don't really think a fly-to member belongs in the viewer, as it's a property that's specific to animation, not the viewer.

File: open-sphere-base/core/src/main/java/io/opensphere/core/viewer/impl/Viewer2D.java [hunk 2]
```diff
@@ -181,6 +177,7 @@ public double getHeading()
     @Override
     public MapContext<DynamicViewer> getMapContext()
     {
+        // HEre
```
Comment (roddyaj) [inline open-sphere-base/core/src/main/java/io/opensphere/core/viewer/impl/Viewer2D.java]: Probably remove this

File: open-sphere-base/core/src/main/java/io/opensphere/core/viewer/impl/Viewer2D.java [hunk 3]
```diff
@@ -672,6 +658,7 @@ public boolean equals(Object obj)
         @XmlJavaTypeAdapter(MutableVector3d.Vector3dAdapter.class)
         public Vector3d getLocation()
         {
+            // here
```
Comment (roddyaj) [inline open-sphere-base/core/src/main/java/io/opensphere/core/viewer/impl/Viewer2D.java]: Probably remove this

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/util/impl/DataElementActionUtilsImpl.java [hunk 4]
```diff
@@ -84,57 +87,39 @@ public boolean gotoSelectedFeature(long deId, boolean flyTo)
...
-            }
-            if (flyTo)
-            {
-                animator.start();
-            }
-            else
-            {
-                animator.snapToPosition();
-            }
+            DataTypeActionUtils.gotoBoundingBox(bbox, view,flyTo);
```
Comment (roddyaj) [inline open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/util/impl/DataElementActionUtilsImpl.java]: nitpick: there should be a space between the comma and flyTo

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/util/impl/DataTypeActionUtils.java [hunk 5]
```diff
@@ -41,16 +41,17 @@ public static void gotoDataType(DataTypeInfo dataType, DynamicViewer viewer)
...
-        gotoBoundingBox(bbox, viewer);
+        gotoBoundingBox(bbox, viewer, true);
     }
 
     /**
      * Goes to the bounding box.
      *
      * @param bbox the bounding box
      * @param viewer the viewer
+     * @param flyTo the flyTo animation
```
Comment (roddyaj) [inline open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/util/impl/DataTypeActionUtils.java]: Let's explain what this is better, like true to fly to, and false to snap to, or something like that.

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/util/impl/DataElementActionUtilsImpl.java (hunk 4 above)
Comment (brelsftm) [reply open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/util/impl/DataElementActionUtilsImpl.java]: oh wait i fixed a different comma mistake , shoot

**Pull request merged**